src/screens/CreatePost/elements/Card.js

```diff
@@ -1,51 +1,52 @@
 import React from 'react';
-import {View, StyleSheet, Platform} from 'react-native';
-import {COLORS} from '../../../utils/theme';
-
-const Card = (props) => {
+import { View, StyleSheet, Platform } from 'react-native';
+import { colors } from '../../../utils/colors';
+const Card = props => {
   return (
     // <View style={{ ...styles.card, ...props.style }}></View>
     <View // Parent
       style={{
         // No backgroundColor
-        shadowColor: COLORS.black,
-        shadowOffset: {width: 0, height: 1},
+        shadowColor: '#000',
+        shadowOffset: { width: 0, height: 1 },
         shadowOpacity: 0.5,
         // shadowRadius: 10,
         zIndex: 999,
         // width: widthInput,
-        ...props.style
-      }}>
+        ...props.style,
+      }}
+    >
       <View // Card
         style={{
           flex: 1,
           borderRadius: 10,
           // To round image corners
           overflow: 'hidden',
-          borderColor: COLORS.silver,
+          borderColor: '#bdbdbd',
           borderWidth: 0.5,
           // https://github.com/facebook/react-native/issues/10049#issuecomment-366426897
-          backgroundColor: COLORS.lightgrey,
+          backgroundColor: colors.lightgrey,
           // Android shadow
           elevation: 4,
           paddingHorizontal: 15,
           paddingVertical: 10,
           width: '100%'
-        }}>
+        }}
+      >
         {props.children}
       </View>
     </View>
   );
 };
 const styles = StyleSheet.create({
   card: {
-    shadowOffset: {width: 10, height: 10},
+    shadowOffset: { width: 10, height: 10 },
     shadowColor: 'black',
     shadowOpacity: 1,
     elevation: 3,
     // background color must be set
-    backgroundColor: COLORS.black, // invisible color
-    zIndex: 999
+    backgroundColor: "#0000", // invisible color
+    zIndex: 999,
   }
 });
-export default Card;
+export default Card;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is no newline at the end of the file. It's a common convention to end files with a newline because it helps with version control systems like Git and can prevent potential issues.

```diff
- export default Card;
+ export default Card;\n
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->